src/Models/Backup.php

```diff
@@ -177,12 +174,11 @@ public function restore()
      */
     public function serialize()
     {
-        $date_format = $this->getConfig()->get('date_format');
         return [
             'file'      => $this->get('filename'),
             'size'      => $this->getSizeInMb(),
-            'date'      => date($date_format, $this->getDate()),
-            'expiry'    => date($date_format, $this->getExpiry()),
```

Comment:
I'm too late for this comment, but I will mention it anyway. It would be better for `serialize` to return the time in epoch format, or perhaps just yyyy-mm-dd-hh-mm form -- anything machine-readable -- and then convert to human-readable format with a cell formatter. That way, scripts asking for json or yml would get easy-to-parse values, while humans using the default table format would get easy-to-read values.

Doing this change now would be a backwards-compatibility break, though. I suppose you could make `date-raw` and `expiry-raw`, etc., but that is probably a decision for another day.